bolt/src/main/java/com/slack/api/bolt/App.java

```diff
@@ -68,6 +72,16 @@
      */
     private final Slack slack;
 
+    /**
+     * The default Web API client without any tokens.
+     */
+    private final MethodsClient client;
+
+    /**
+     * The built-in handy way to run operations asynchronously. It's totally fine to use your own one instead.
+     */
+    private final ExecutorService executorService;
```

Comment:
In Java, we should enable developers to use their own runtime for async executions. That said, a built-in handy way is helpful for many. So, we introduce ExecutorService as an option since v1.2.